### PR TITLE
Add sca scan docs and CVEs to release notes

### PR DESCRIPTION
**Pull request by julia-pu**
docs: [META-366][META-298] add sca scan results in scst-store docs and point release note CVE section to CVE.md

### CODE REVIEW

File: scst-store/release_notes.md [hunk 1]
```diff
@@ -2,6 +2,45 @@
 
 This topic contains release notes for Supply Chain Security Tools - Store.
 
+## v1.0.0-beta.2
+
+**Release Date**: November 26, 2021
```
Comment (Manifaust) [inline scst-store/release_notes.md]: The release date is actually when TAP beta 4 is released altogether. That's when the public can see it.

Anyways you can probably remove this.

File: scst-store/sca_scans/CVE.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* Grype
+
+## Version of Source:
+
+* BDBA version SDL 4.2
+* Grype version 0.25.1
+
+## CVEs
+### BDBA
+No vulnerabilities found in both the binaries of metadata-store and CLI.
```
Comment (Manifaust) [inline scst-store/sca_scans/CVE.md]: In the official docs, we're not using the term metadata-store, or even _The Store_. In this case it's easiest to call it API backend.

File: scst-store/release_notes.md [hunk 3]
```diff
@@ -2,6 +2,45 @@
...
+
+See [Troubleshooting and Known Issues](known_issues.md).
+
+### Limitations
+
+* Air gapped environments are not supported
+
+### CVEs
+
+See [SCA Scanning Results](sca_scans/CVE.md)
```
Comment (miketeve) [inline scst-store/release_notes.md:43]: Can we also mention (1) no criticals or highs and (2) that we scanned each of our components with black duck and grype

File: scst-store/sca_scans/CVE.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# SCA Scanning Results
+
```
Comment (miketeve) [inline scst-store/sca_scans/CVE.md:4]: Add date?  I know it's in the file name but perhaps good to include it here as well

File: scst-store/release_notes.md (hunk 1 above)
Comment (julia-pu) [reply scst-store/release_notes.md]: okay, removed

File: scst-store/sca_scans/CVE.md (hunk 2 above)
Comment (julia-pu) [reply scst-store/sca_scans/CVE.md]: Changed to "API backend"

File: scst-store/release_notes.md (hunk 3 above)
Comment (julia-pu) [reply scst-store/release_notes.md:43]: Okay added!

File: scst-store/sca_scans/CVE.md (hunk 4 above)
Comment (julia-pu) [reply scst-store/sca_scans/CVE.md:4]: Added

**Review by Manifaust — APPROVED** [review]

**Pull request merged**